src/clientSideScene/segments.ts

```diff
@@ -686,19 +686,20 @@ class CircleSegment implements SegmentUtils {
 
 export function createProfileStartHandle({
   from,
-  id,
-  pathToNode,
+  isDraft,
   scale = 1,
   theme,
   isSelected,
+  ...rest
 }: {
   from: Coords2d
-  id: string
-  pathToNode: PathToNode
   scale?: number
   theme: Themes
   isSelected?: boolean
-}) {
+} & (
+  | { isDraft: true }
+  | { isDraft?: false; id: string; pathToNode: PathToNode }
```

Comment:
`isDraft?: false` specifically the `?:` smells a bit to me.

Reply:
Okay no worries I've removed it. It just means you can set it but it has to be `false` if you're going to also pass in `pathToNode` as well.

Reply:
Having a default arg value probably feels better?